docker-compose.yml

```diff
@@ -37,6 +44,12 @@ services:
       - DJANGO_ENV
       - USGS_USERNAME
       - USGS_PASSWORD
+      - PANOPTES_PROD_CLIENT_ID
```

Comment:
I mentioned this in an earlier PR but these should be converted to remove the deploy environment `PANOPTES_CLIENT_ID`, you can easily switch the targets when running docker-compose up by changing these env var values or overriding them on the cmd line.

Reply:
When I inherited theia, she could only connect to production version of panoptes. Not staging. I would like to get to the point where she can also upload subjects to staging for testing out changes without affecting data on prod, and I think we're almost there.

In the meantime, I have this set up so that I can switch between pointing at projects on prod Panoptes and staging Panoptes with my local theia, and not have to keep copy and pasting over the keys as I am developing.

For deployment, we can collapse them.

Reply:
yep - i get it, i'd just modify the env var values to switch the target API rather than have both available and a pivot boolean / flag.